lib/cmd/install.cpp

```diff
@@ -1,4 +1,5 @@
 #include <cstdlib>
+#include <filesystem>
```

Comment:
remove this line, it's already export via `util/fs.h`

Reply:
removed unused <thread> and "cppship/util/repo.h" too.